kernel/src/process.rs

```diff
@@ -1485,11 +1496,13 @@ impl<'a> Process<'a> {
 \r\n             │ Unused\
   \r\n  {:#010X} ┴───────────────────────────────────────────\
 \r\n             .....\
-  \r\n  {:#010X} ┬─────────────────────────────────────────── F\
-\r\n             │ App Flash    {:6}                        L\
-  \r\n  {:#010X} ┼─────────────────────────────────────────── A\
-\r\n             │ Protected    {:6}                        S\
-  \r\n  {:#010X} ┴─────────────────────────────────────────── H\
+  \r\n  {:#010X} ┬───────────────────────────────────────────\
+\r\n             │ App Flash    {:6}                        F\
+  \r\n  {:#010X} ┼─────────────────────────────────────────── L\
+\r\n             │ App State    {:6}                        A\
+  \r\n  {:#010X} ┼─────────────────────────────────────────── S\
+\r\n             │ Protected    {:6}                        H\
+  \r\n  {:#010X} ┴───────────────────────────────────────────\
```

Comment:
This might be fine, but again makes assumptions about the app and how it laid out its binary.

Reply:
We could pull this out if there's no guarantee of app_state placement. I would just have to ensure that the app_flash region and address range includes the writeable region, if it exists.